src/editor/graph/actions/default_action_registrar.cpp

```diff
@@ -30,7 +30,9 @@ void OrchestratorDefaultGraphActionRegistrar::_register_node(const OrchestratorG
                                                        const Dictionary& p_data)
 {
     OScriptLanguage* language = OScriptLanguage::get_singleton();
-    Ref<OScriptNode> node = language->create_node_from_name(p_class_name, p_context.script, false);
+    Orchestration* orchestration = p_context.graph->get_orchestration();
+
+    Ref<OScriptNode> node = language->create_node_from_name(p_class_name, orchestration, false);
```

Comment:
```suggestion
    Ref<OScriptNode> node = p_context.graph->create_node_from_name(p_class_name, false);
```
Can we abstract this behind the `Ref<OScriptGraph>` for future expansion needs.